group_members.go

```diff
@@ -28,18 +28,30 @@ type GroupMembersService struct {
 	client *Client
 }
 
+// GroupMemberSamlIdentity represents the SAML Identity link for the group member.
+//
+// GitLab API docs: TBD
+// Gitlab MR for API change: https://gitlab.com/gitlab-org/gitlab/-/merge_requests/20357
+// Gitlab MR for API Doc change: https://gitlab.com/gitlab-org/gitlab/-/merge_requests/25652
+type GroupMemberSamlIdentity struct {
```

Comment:
In Go initialisms or acronyms are always written in al caps: https://github.com/golang/go/wiki/CodeReviewComments#initialisms

So for example `GroupMemberSamlIdentity` should be `GroupMemberSAMLIdentity` and `SAMLProviderID` should be `SAMLProviderID`.

Reply:
Changed.